content/docs/gke/gcp-e2e.md

```diff
@@ -840,17 +746,17 @@ minutes for the IP address to be available:
     kubectl get service web-ui
     ```
 
-1. Copy the value shown under `EXTERNAL-IP` and paste it into your web
+2. Copy the value shown under `EXTERNAL-IP` and paste it into your web
```

Comment:
Please revert this change, and all the similar changes from 1. to a specific number below. We keep the list formatting as just 1. to avoid having to renumber when we add/remove steps.

Reply:
That's strange, I did not update that, may be updated automatically by editor I think :-)